xfer/publisher.go

```diff
@@ -83,12 +94,73 @@ func (p HTTPPublisher) Publish(rpt report.Report) error {
 	return nil
 }
 
+// Stop implements Publisher
+func (p HTTPPublisher) Stop() {}
```

Comment:
File a bug for it; this change is not a regression against the existing codebase, but we should do it in the future.